crates/autopilot/src/on_settlement_event_updater.rs

```diff
@@ -171,6 +171,18 @@ impl Inner {
 
         tracing::debug!(?hash, ?update, "updating settlement details for tx");
 
+        {
+            // temporary to debug and compare with current implementation
+            // TODO: use instead of current implementation
+            let settlement =
+                domain::settlement::Tx::new(hash.into(), &self.eth, &self.persistence).await;
+            tracing::info!(?settlement, "settlement object");
```

Comment:
nit: log message is redundant

Reply:
I intented to use it for log search... typing just `settlement` might not hit what I want.